staging/src/kubevirt.io/client-go/api/v1/schema.go

```diff
@@ -355,6 +355,11 @@ type Devices struct {
 	// Whether to attach the default serial console or not.
 	// Serial console access will not be available if set to false. Defaults to true.
 	AutoattachSerialConsole *bool `json:"autoattachSerialConsole,omitempty"`
+	// Whether to attach the Memory balloon device with default 10s period.
+	// Period can be also set in virt-config.
```

Comment:
I would not mention the default period here, but I would instead include the exact name of the ConfigMap attribute that can be used to tweak it.